mesonbuild/interpreter/compiler.py

```diff
@@ -734,3 +743,57 @@ def get_supported_function_attributes_method(self, args: T.Tuple[T.List[str]], k
     @noKwargs
     def get_argument_syntax_method(self, args: T.List['TYPE_var'], kwargs: 'TYPE_kwargs') -> str:
         return self.compiler.get_argument_syntax()
+
+    @typed_kwargs('compiler.symbol_visibility_header',
+        KwargInfo('api', (str, NoneType)),
+        KwargInfo('compilation', (str, NoneType)),
+        KwargInfo('static_compilation', (str, NoneType)),
+        KwargInfo('static_only', bool, default=False),
+        )
+    @typed_pos_args('compiler.symbol_visibility_header', str)
+    @FeatureNew('compiler.symbol_visibility_header', '0.63.0')
+    def symbol_visibility_header_method(self, args: T.Tuple[str], kwargs: 'SymbolVisibilityHeaderKW') -> mesonlib.File:
+        if self.compiler.language not in {'c', 'cpp'}:
```

Comment:
Will this also work for Objective-C / Objective-C++?

Reply:
I don't know how it works with objc, but if someone who knows can confirm it works, it can be added there, yes.

Reply:
It works the same, Objective-C is a superset of C, so all you can do there, you can in Objective-C too.

Reply:
Ok, updated.